iree/compiler/Codegen/Common/SetNumWorkgroupsPass.cpp

```diff
@@ -21,6 +21,39 @@ namespace mlir {
 namespace iree_compiler {
 
 namespace {
+/// Sets the hal.interace.workgroup.size operation to the constant value passed
```

Comment:
This shouldnt be needed. `materializeStaticLaunchInformation` does exactly this.